pkg/features/helpers.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
```

Comment:
Why is there only core API's helpers in this folder?

Reply:
Haven't looked into the other groups. I would like to make the pkg/apis packages independent from features (with the exception of validations). In general, the groups implementation should be as self-contained as possible, like a library on-top of k8s.io/api. If we have similar handlers in other groups, we move them here as well and potentially promote the helpers.go to another structure.

Reply:
Ok. Please ping me on slack when this needs another review.

Reply:
Just have to rebase again. Rebase hell...